src/App.js

```diff
@@ -1,23 +1,19 @@
-import logo from './logo.svg';
+import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import Home from './views/Home/Home';
 import './App.css';
+import Header from './components/Header/Header';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
```

Comment:
Unnecessary div. If you're only using it to return multiple elements, then a fragment will do just fine without rendering a DOM node. It seems like a minor thing but can quickly lead to div soup if you're not careful.